pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.3.0"
+    version: "3.1.6"
```

Comment:
I really don't know why the lock file keeps changing...

Reply:
It's because it's auto generated based on the content of the rest of the project